src/features/messages/speakQueue.ts

```
@@ -15,6 +15,19 @@ export class SpeakQueue {
   private queue: SpeakTask[] = []
   private isProcessing = false
   private currentSessionId: string | null = null
+  private static speakCompletionCallbacks: (() => void)[] = []
+
+  // 発話完了時のコールバックを登録
+  static onSpeakCompletion(callback: () => void) {
+    this.speakCompletionCallbacks.push(callback)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)